src/api/app/services/scm_status_reporter.rb

```diff
@@ -26,6 +28,8 @@ def call
                                          @state,
                                          status_options)
     end
+  rescue StandardError => e
+    rescue_with_handler(e) || raise(e)
   end
```

Comment:
```suggestion
  rescue Octokit, Gitlab::Error => e
    rescue_with_handler(e)
  end
```

Reply:
@hennevogel the rescue exceptions list has been updated but we still need to use this `raise(e)`. `rescue_with_handler` apparently ignores every exception that is not mentioned in the rescue statement.